tests/testthat/test-load.R

```diff
@@ -28,40 +28,42 @@ test_that('user commands fail when not in ProjectTemplate directory', {
 
         # Check cache()
         expect_error(cache())
-        
+
         # Check reload.project()
         expect_error(reload.project())
-        
+
         # Check reload.project()
         expect_error(test.project())
-        
+
         # Check stub.tests()
         expect_error(stub.tests())
-        
+
         # Check project.config()
         expect_error(project.config())
-        
+
 })
 
 test_that('auto loaded data is cached by default', {
         test_project <- tempfile('test_project')
         suppressMessages(create.project(test_project, minimal = FALSE))
         on.exit(unlink(test_project, recursive = TRUE), add = TRUE)
-        
+
         oldwd <- setwd(test_project)
         on.exit(setwd(oldwd), add = TRUE)
-        
-        
+
+        # clear the global environment
+        rm(list=ls(envir = .TargetEnv), envir = .TargetEnv)
```

Comment:
I copied some code from the other tests since that test started failing because it assumed the environment was clear instead of making sure it was clear by itself. `clear` is better indeed, I will review the other tests as well.